lib/double_entry/reporting.rb

```diff
@@ -142,28 +142,6 @@ def aggregate_array(function:, account:, code:, partner_account: nil, filter: ni
                          filter: filter, range_type: range_type, start: start, finish: finish)
     end
 
-    # Identify the scopes with the given account identifier holding at least
-    # the provided minimum balance.
-    #
-    # @example Find users with at least $1,000,000 in their savings accounts
-    #   DoubleEntry::Reporting.scopes_with_minimum_balance_for_account(
-    #     1_000_000.dollars,
-    #     :savings,
-    #   ) # might return the user ids: [ 1423, 12232, 34729 ]
-    # @param [Money] minimum_balance Minimum account balance a scope must have
-    #   to be included in the result set.
-    # @param [Symbol] account_identifier
-    # @return [Array<Integer>] Scopes
-    #
-    def scopes_with_minimum_balance_for_account(minimum_balance, account_identifier)
```

Comment:
It looks like the new place makes more sense. Do we need to keep a method here for backwards compatibility?

Reply:
That's a great question @scottyp-env. This `Reporting` module is marked @api private and is not yet considered stable. I think it's ok to simply move it.

Reply:
Perfect!